response_operations_ui/templates/edit-collection-exercise-details.html

```diff
@@ -0,0 +1,104 @@
+
+<html dir="ltr" lang="en-gb">
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+
+  <title>Edit collection exercise details</title>
+
+  <link rel=stylesheet type=text/css href="https://cdn.ons.gov.uk/sdc/322aece/css/styles.css">
+  {% assets "scss_all" %}
+  <link rel=stylesheet type=text/css href="{{ ASSET_URL }}">
+  {% endassets %}
+
+  <link href="/static/images/favicon.ico" rel="Shortcut Icon" />
+
+</head>
+
+<body class="spoke">
+
+<div class="page">
+
+  <div class="page__content">
+
+    <div class="container page__container">
+
+      <div class="grid">
+        <div class="grid__col col-3@l">
+        </div>
+
+        <div class="grid__col col-6@l">
+
+          <div role="main" id="main" class="page__main">
+
+
+            <div class="spoke__header">
+
+              <div class="spoke__header__title">
+
+                <h1 class="jupiter">Edit collection exercise details</h1>
+
+              </div>
+
+            </div>
+
+            <div class="spoke__body">
+
+              <form action="" method="post">
+                {{ form.csrf_token }}
+                <div class="field u-mb-l">
+
+                  <label class="label" for="Survey">
+                    Survey:
+                    <span class="u-vh">Current value: </span>
+                      <span class="mars">
+                        {{ survey_ref }} {{ short_name }}
+                      </span>
+                  </label>
+
+
+                  <label class="label" for="Period">
+                  Period:
+                    <span class="u-vh">Current value: </span>
+                      <span class="mars">
+                      {{ period }}
+                      </span>
+                    </label>
+
+                  <br>
+
+                  <label class="label">
+                    Period as shown to respondent
+                    <span class="u-vh">Current value: </span>
+                    {{ user_description }}
+                    </span>
+                  </label>
+                  {{ form.user_description(class_="input input--text", id='user_description', required=required, maxlength=254) }}
```

Comment:
The current `Period as shown to respondent` value should appear in the input box when the page loads